_data/domains.yml

```
@@ -207,7 +207,7 @@ websites:
       tfa: Yes
       software: Yes
       hardware: Yes
-      doc: https://doc.gandi.net/en/organization/users/security
+      doc: https://docs.gandi.net/en/account_management/security/index.html
```

To make it less likely to break, how about just removing the `index.html` part?
```suggestion
      doc: https://docs.gandi.net/en/account_management/security/
```